scout/server/blueprints/variants/templates/variants/mei-variants.html

```diff
@@ -92,7 +92,7 @@
   {% if variant.dismiss_variant %}
    <tr class="dismiss">
   {% elif 'causatives' in case and variant._id in case.causatives %}
-   <tr class="causative">
+   <tr class="causative">v
```

Comment:
```suggestion
   <tr class="causative">
```

Reply:
Ahem, really, I shouldn't be checking in that file at all.